helix-view/src/theme.rs

```diff
@@ -157,6 +157,16 @@ impl Theme {
         self.styles.get(scope).copied()
     }
 
+    /// Get the style of a scope, falling back to dot separated broader
+    /// scopes. For example if `ui.text.focus` is not defined in the theme,
+    /// `ui.text` is tried and then `ui` is tried. If none of the fallbacks
+    /// are found, the default style is returned.
+    pub fn get_with_fallback(&self, scope: &str) -> Style {
```

Comment:
Should this just be the default? get/try_get is already used for UI scopes only and we try to reuse the value after lookup. This would seamlessly make a lot of ui scopes nicer to use since it would have proper fallback

Reply:
That's a good idea; done in 9aaafc7.